src/cargo.rs

```diff
@@ -212,17 +211,13 @@ pub fn config_get(key: &str) -> Result<String> {
         anyhow!(
             "Command failed with status: {:#?} and stderr: {:#?}.",
             out.status,
-            OsStr::from_bytes(&out.stderr),
+            String::from_utf8(out.stderr).unwrap(),
```

Comment:
In order to avoid the `unwrap`, I would prefer either

```suggestion
            String::from_utf8(out.stderr)?,
```

or simply

```suggestion
            String::from_utf8(out.stderr),
```

to keep context.